enhancements/scorecard-metadata.md

```diff
@@ -0,0 +1,115 @@
+---
+title: scorecard-metadata
+authors:
+  - "@jemccorm"
+  - "@jlanford"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-04-29
+last-updated: 2020-05-14
+status: provisional
+---
+
+# scorecard-metadata
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+This enhancement is to define a place within the bundle format whereby
+scorecard metadata could be added.  This will allow tests to be co-located
+with the normal operator content found in a bundle.  This enhancement
+suggests naming and paths for the scorecard metadata within the bundle.
+
+## Motivation
+
+The purpose of this enhancement is to define a location for scorecard
+and other test assets to be included into the bundle.  This definition
+informs test frameworks and tools of a well-known location to search for test assets
+within a bundle and reduces confusion to test developers as to 
+where test assets will reside.
+
+### Goals
+
+This enhancement will be used by the scorecard feature within the operator-sdk
+initially as part of a major redesign of scorecard functionality.  Other
+test frameworks apart from operator-sdk or scorecard could leverage this
+enhancement as well.
+
+### Non-Goals
+
+This enhancement is scoped to define the location within the bundle for
+this sort of metadata and get consensus on names and paths.  Follow on
+work would be required to implement tooling that lets a user inject
+their test assets into a bundle image as well as extract this metadata.
+
+## Design Details
+
+The bundle format would allow for test metadata to be
+included in the following locations within a bundle:
+
+```
+bundle/
+├── manifests
+│   ├── cache.example.com_memcacheds_crd.yaml
+│   └── memcached-operator.clusterserviceversion.yaml
+├── metadata
+│   └── annotations.yaml
+└── tests
+    └── scorecard
+        ├── config.yaml
+        └── kuttl
+            ├── example1
+            └── example2
+```
+
+In the above proposed layout, the test metadata resides under a new
+directory called `tests`.  Below that directory is free form but
+in the case of operator-sdk scorecard, it would populate this
+directory with a subdirectory named `scorecard` as depicted.
+
+The scorecard test label annotations for the bundle would be as follows:
+```
+annotations:
+  operators.operatorframework.io.test.mediatype.v1: "scorecard+v1"
+  operators.operatorframework.io.test.config.v1: "/tests/scorecard/config.yaml"
+  # optional
+  operators.operatorframework.io.test.type.v1: "kuttl"
```

Comment:
What does `operators.operatorframework.io.test.type.v1: "kuttl"` convey and how would we act on it?